### ISSUE TITLE
PWA - Add installation prompt to install the website as an app on desktop and on mobile devices

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3695]
Add installation prompt to install the website as an app on desktop and on mobile devices - add PWA to website on all devices.

@MendyBerger do you need to have a design of where to put the **install** button? Should @GaliShapira prepare the design for you yet?

**Comment by MendyBerger** [comment]
Yes. But keep in mind that it won't work on some browsers, so the page has to look good without the install button as well.
But I can start the work without the design, adding the install button will be that last part.
